frontend/pages/request.tsx

```diff
@@ -88,7 +88,7 @@ const RequestSplit: NextPage = () => {
         )
```

Comment:
`this comment refers to line 69`
Can replace the useEffect hook with useStakeAccounts hook.